### PR TITLE
Fixes #65 Deploy the app to apk on pull request merge

### LINKED ISSUE #65: Travis tried to deploy the app to apk on pull requests
Travis credentials for the repo is needed to push the generated apk to the apk branch, since they are not available to pull requests, we could run these scripts on direct commits, since github addresses PR merge commits like direct commit, we can safely have the apk generated for PR merges.

This issue is very easy and can be solved by a one line solution with the help of a Travis variable. 

This would improve the build time for PR, as Travis currently tries to push after cloning the repo and it can't push and there is no need to do so!

### PR DESCRIPTION
**Pull request by anantprsd5**
Fixes issue #65 Travis tried to deploy the app to apk on pull requests

Changes: Now apk will build on direct commits.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -40,8 +40,10 @@ script:
  - ./gradlew build
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash)
-  - ./upload-apk.sh
+    - if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+        bash <(curl -s https://codecov.io/bash)
```
Comment (Abhi2424shek) [inline .travis.yml]: Pls let codecov run on PR make this line above the if condition pls

Comment (anantprsd5) [inline .travis.yml]: Okay. Making changes

Comment (Abhi2424shek) [inline .travis.yml]: Sure

**Comment by anantprsd5** [comment]
@Abhi2424shek Pls have a look now.

File: .travis.yml [hunk 2]
```diff
@@ -40,8 +40,10 @@ script:
  - ./gradlew build
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash)
-  - ./upload-apk.sh
+    - bash <(curl -s https://codecov.io/bash)
+    - if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
```
Comment (Abhi2424shek) [inline .travis.yml]: Give it quotes ' (single) from the if in the begining to the fi in the end and make it one line if possible pls

Comment (anantprsd5) [inline .travis.yml]: Done

**Comment by Abhi2424shek** [comment]
Thanks

**Pull request merged**